src/devices/bus/mc10/mc10_cart.h

```diff
@@ -91,13 +91,13 @@ class device_mc10cart_interface : public device_interface
 	// construction/destruction
 	virtual ~device_mc10cart_interface();
 
-	virtual int max_rom_length();
+	virtual const int max_rom_length();
```

Comment:
That’s not going to do anything – `const` on the return type when returning by value isn’t part of the function signature at all.  It’s `T foo(...) const` to const-qualify a member function.